### ISSUE TITLE
[Bug] Exception: Symfony\Component\HttpKernel\Exception\HttpException

### ISSUE THREAD

**Issue by marionassef** [Qoraiche/laravel-mail-editor#84]
Hey there 
I am using 
1- Laravel 6.2 
2- this package 1.2

when I call php artisan route:list 
I got Symfony\Component\HttpKernel\Exception\HttpException
from qoraiche\mailEclipse\Http\Controllers\MailablesController@toMailablesList

any help will be appreciated
thanks

**Comment by 4n70w4** [comment]
```
php artisan route:list
```


```
   Symfony\Component\HttpKernel\Exception\HttpException  :

  at /var/www/app/vendor/laravel/framework/src/Illuminate/Foundation/Application.php:1014
    1010|         if ($code == 404) {
    1011|             throw new NotFoundHttpException($message);
    1012|         }
    1013|
  > 1014|         throw new HttpException($code, $message, null, $headers);
    1015|     }
    1016|
    1017|     /**
    1018|      * Register a terminating callback with the application.

  Exception trace:

  1   Illuminate\Foundation\Application::abort()
      /var/www/app/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php:46

  2   abort()
      /var/www/app/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php:87
```


workaround: delete `abort_unless` from `\qoraiche\mailEclipse\Http\Controllers\MailablesController::__construct` and `\qoraiche\mailEclipse\Http\Controllers\TemplatesController::__construct`

**Comment by peyobr** [comment]
+1

**Comment by ReeceM** [comment]
This is resolved with the latest versions `v3.2` and `v2.4`
